lib/voicebase/v2/client.rb

```diff
@@ -79,6 +133,22 @@ def get_media_progress(args = {}, headers = {})
         ), api_version)
       end
 
+      def delete_file(args = {}, headers = {})
```

Comment:
Could also put the ArgumentError right at the top to make it's requiredness jump out a bit more:

```ruby
def delete_file(args = {}, headers = {})
  raise ArgumentError, "Missing argument :media_id" unless args[:media_id]

  url = uri + "/media/#{args[:media_id]}"
```